server/index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const calendar = require('express').Router();
+const path = require('path');
+const app = express();
+const getCalendar = require('./calendar');
+const port = process.env.PORT || 3030;
+
+
+calendar.get('/api/calendar', getCalendar);
+app.use('/api/calendar', getCalendar),
+app.use('/', express.static(path.join(__dirname, '../dist')))
```

Comment:
Lines 11 and 12 mean exactly the same with following difference:
- 11 adds handler to `calendar` router
- 12 adds handler to global router ie `app`

That's redundant and besides that you are not even using `calendar` anywhere.